pkg/kubelet/container_bridge.go

```diff
@@ -27,8 +28,39 @@ import (
 
 var cidrRegexp = regexp.MustCompile(`inet ([0-9a-fA-F.:]*/[0-9]*)`)
 
+func createCBR0(wantCIDR *net.IPNet) error {
+	// recreate cbr0 with wantCIDR
+	if err := exec.Command("brctl", "addbr", "cbr0").Run(); err != nil {
+		glog.Error(err)
+		return err
+	}
+	if err := exec.Command("ip", "addr", "add", wantCIDR.String(), "dev", "cbr0").Run(); err != nil {
+		glog.Error(err)
+		return err
+	}
+	if err := exec.Command("ip", "link", "set", "dev", "cbr0", "up").Run(); err != nil {
+		glog.Error(err)
+		return err
+	}
+	// restart docker
+	if err := exec.Command("service", "docker", "restart").Run(); err != nil {
```

Comment:
@kelseyhightower good point. But systemctl only covers part of os distro, not rest of them. Besides init.d and systemd, we have os distro which is chrome os based is using upstart. This is part of NodeSpec work I mentioned and want to put more efforts on after post 1.0.

For now, I think what @thockin suggested should work here.